go/test/endtoend/vtgate/queries/informationschema/informationschema_test.go

```diff
@@ -206,3 +203,24 @@ func TestMultipleSchemaPredicates(t *testing.T) {
 	require.Error(t, err)
 	require.Contains(t, err.Error(), "specifying two different database in the query is not supported")
 }
+
+func TestInfrSchemaAndUnionAll(t *testing.T) {
+	clusterInstance.VtGateExtraArgs = append(clusterInstance.VtGateExtraArgs, "--planner-version=gen4")
+	require.NoError(t,
+		clusterInstance.RestartVtgate())
```

Comment:
We should change the planner version of VTGate back to its original value once this test is finished. Otherwise the other tests will only be executed using the `Gen4` planner. Otherwise, we could simply use the planner version comment directive.

Reply:
I thought about that. but does not see that as an important change. Any new test added to the file can change if seems required.